src/isomorphic/React.js

```diff
@@ -129,8 +135,10 @@ if (__DEV__) {
         lowPriorityWarning(
           false,
           'Accessing factories like React.DOM.%s has been deprecated ' +
-            'and will be removed in the future. Use the ' +
-            'react-addons-dom-factories package instead.',
+            'and will be removed in v16.0+. Use the ' +
+            'react-addons-dom-factories package instead. ' +
```

Comment:
Thanks - going to do an overall clean-up of that. The README in that package has the wrong name, and probably other places too.